### ISSUE TITLE
Info Dialogs

### ISSUE THREAD

**Issue by bartolomesoriano** [mistervee/skin.madnox#6]
_From @mistervee on April 26, 2017 2:36_

INFO DIALOGS	
DialogAddonInfo.xml
DialogMusicInfo.xml	
DialogVideoInfo.xml	
DialogSongInfo.xml	Removed (not used since jarvis)
DialogPictureInfo.xml
DialogPlayerProcessInfo.xml

_Copied from original issue: mistervee/skin.madnox.backup#5_

**Comment by meowmoo** [comment]
DialogAddonInfo
option a; https://drive.google.com/drive/folders/0B9bSf6HADVkFSGJ5aGoxcVFMa3M
option b; https://drive.google.com/open?id=0B9bSf6HADVkFRDVpUzM5eGhHWGc

**Comment by bartolomesoriano** [comment]
@meowmoo, I like option a. Well done!

**Comment by meowmoo** [comment]
I like b more :P

**Comment by bartolomesoriano** [comment]
Submit option b, then 😊

**Comment by mistervee** [comment]
@meowmoo 
Tough call on that. I like option b, but some addons don't have fanart so that might look a bit strange with the empty space below the buttons. Option a on the other hand will look better if there is no addon fanart.

All the listed xml's in the first post provide info for an item. If you look at mimic or estuary they are all similar in design giving a consistent look when pressing the info button. Also they cover the fullscreen. So i am thinking that we should get some sort of basic design that will be similar between the dialogs.

Thoughts?

**Comment by bartolomesoriano** [comment]
@mistervee, that makes a lot of sense.  How do we go about it? I envision we would have an include like the one for the other dialogs that we would use for the info ones?

**Comment by bartolomesoriano** [comment]
I am terrible with creative stuff, will either @mistervee or @meowmoo take a stab at a mock up?

**Comment by mistervee** [comment]
I am having some issues with the creative side also. Really slowing coding down. 

But its to be expected, as i think we are still ironing out some kinks with the overall look.

This is a good site for inspiration
https://au.pinterest.com/pin/337136722076434109/

**Comment by bartolomesoriano** [comment]
Thanks @mistervee! What do you all think of these to take elements from in deciding our info dialog look for movies and tv? http://imgur.com/a/k8eBq

@meowmoo, @mistervee, @MadMikeDoc

**Comment by meowmoo** [comment]
ok, so we are going full screen!?
DialogMusicInfo.xml
DialogVideoInfo.xml
should be full screen like it is in the Old Madnox, i will give AddonInfo another try and see what I cam up with :)

**Comment by meowmoo** [comment]
I moved around a few things, and push option b, so you can try it. its not polished ;)

my 2 cent here:

Music/video info will always stay out layout wise, because there is much more info we can use and much more/better Art.
so why not totally break the consistent look, with a Dialog for AddonInfo and a VideoOSD like (metadata instead of buttons) PictureInfo 
in the dialog is all the Info you need for the addon, its clean and simple, if we go fullscreen this little info will look kinda lost in my eyes, or i need to stare at the "ugly" addon Icon because its so big.

not 100% on DialogPVRInfo but with skinhelper there should be more info/art so we can for the Music/video Look

**Comment by meowmoo** [comment]
@barts link, personally I would use spiderman 2, but most users want something like mad max.

Well its MADnox add both xD

**Comment by bartolomesoriano** [comment]
@meowmoo, I agree with your opinion on AddonInfo, I actually have never used photos with Kodi, so I can't comment about PictureInfo.

I like the idea of something like the Spiderman 2 example for video info, with buttons on the top, but I also enjoy having a lot of info, so perhaps that can be options you can toggle? I like being able to see the cast info, and having media flags, logo as title, CD art, etc.  And, especially important for me, audio language and subtitle flags.  The thing is, what would be an attractive layout that doesn't look cluttered and looks elegant? That's beyond my creative abilities! xD

**Comment by mistervee** [comment]
@meowmoo @bartolomesoriano 
I am liking the approach to a simple addon info. My thoughts would be to put the buttons to the left of the image vertical. They seem a little to spread out horizontal.

On the side of music/video info, the biggest issue i have with madnox krypton is the music info. It looks a little out of place with the album artwork and logo. I like the spiderman one myself. Just need to add media flags and disc art. My thoughts would be to have the only a few buttons in the middle of the screen. Any extra info like cast, similar movies extra detail could just be simple as navigating down, so just toggling between views.

I have been working on the vertical menu but a little stalled on some aspects, so i will update the video info.

**Comment by MadMikeDoc** [comment]
I like what we have / had now, but I guess spending all my time creating / enhancing what we had in Krypton is a factor on my view.

**Comment by bartolomesoriano** [comment]
@MadMikeDoc, I like it too, but I think it is difficult to maintain, isn't it? I guess we can arrive at having all of the functionality we all love from MadNox with more efficient code that we can maintain without too many headaches :)

**Comment by mistervee** [comment]
@bartolomesoriano 
I would look at it as inefficient coding. The changes over the years to the skinning engine since nox 4 was released has made it that way. The main thing to consider is how much different do we need to be from a graphic design side to separate this from aeon.

A quote from the wiki (for repo submission)
'No mods. Mods are great, and pop up frequently in the forums. But they are only confusing to non-forum-goers. This can be vague, each is evaluated on its own merit. By naming your "mod" different and make it substantial different enough, we will of course take it in consideration."
Add-on_rules#Requirements_for_skins (wiki)

**Comment by ArieS1204** [comment]
Hey all, 
I'm here to help out on the gfx side but to be honest, anything else is Chinese to me :\
So I'm happy to give a try to anything you ask me to but you have to be specific, think of me as a 5 year old :P

**Comment by MadMikeDoc** [comment]
@ArieS1204 welcome to the team buddy.

**Comment by ArieS1204** [comment]
Thanks Mike, hope I can help.

**Comment by bartolomesoriano** [comment]
@aries1204 welcome! I am pretty sure you will be of great help.

**Comment by meowmoo** [comment]
again, there is a; https://drive.google.com/open?id=0B9bSf6HADVkFWk5XVFYzQkdmaHc
or b; https://drive.google.com/open?id=0B9bSf6HADVkFUE5mNFZCdGhBTUU

im not really happy with both, anyone got a idea :>

**Comment by mistervee** [comment]
Option a looks too squashed.

Option b looks great. My suggestion would be to make it a little wider. So make it the same width as the vertical menu. I haven't finish or pushed the commit for the updated vertical menu, the width is 550 for the background + whatever the white-space is of the texture. For the file path you will need to add ```<haspath>true</haspath>``` to the label.

I don't think there is anything else that could be done to improve on the look as it is just a list of text.

**Comment by meowmoo** [comment]
ok, done
https://drive.google.com/open?id=0B9bSf6HADVkFc3ZNRlpmZGJRZXM

anything to show from the VideoInfo @mistervee, I wanna do the Music Info than :)
